middleware/proxy/README.md

```diff
@@ -121,7 +128,10 @@ type Config struct {
 	WriteBufferSize int
 
 	// tls config for the http client
-	TlsConfig *tls.Config
+	TlsConfig *tls.Config 
+	
+	// Proxy specifies an http proxy to dial http client requests through 
+	Proxy string
```

Comment:
Please use this on Balancer and new test case for this.

Reply:
I add a case  `Test_Proxy_Balancer_WithHttpProxy`

Reply:
thanks